.github/scripts/get_tutorials_stats.py

```diff
@@ -208,20 +221,34 @@ def main() -> None:
             "lines_added": int(i[5]),
             "lines_deleted": int(i[6]),
         }
-        for i in get_history_log
+        for i in tutorials_history_log + pytorch_docs_history_log
     ]
+
+    # Combine the two commits to files
+
+    filenames = []
+    for entry in tutorials_commits_to_files:
+        items = convert_to_dict(entry)
+        for item in items:
+            item["filename"] = f"tutorials/{item['filename']}"
+        filenames.extend(items)
+
+    for entry in pytorch_docs_commits_to_files:
+        items = convert_to_dict(entry)
```

Comment:
just a heads up that the file path here ends up being docs/file, with no mention of pytorch, don't know if that is intended

Reply:
this should work I blieve